src/cpu/o3/store_set.hh

```diff
@@ -149,16 +158,14 @@ class StoreSet
 
     /** Store Set ID Table size, in entries. */
     int SSITSize;
+    int SSITAssoc;
+
+    BaseReplacementPolicy *SSITReplPolicy;
```

Comment:
These 2 variables belong in the cache, not the user of the cache. Remove them from here